src/views/NotesPage.vue

```diff
@@ -98,6 +127,15 @@ export default {
         .catch(error => {
           alert(error);
         });
+    },
+    addTag(newTag) {
+      const vm = this;
+      const tag = {
+        name: newTag,
+        code: newTag.substring(0, 2) + Math.floor(Math.random() * 10000000)
```

Comment:
What's the purpose of `code`?

Reply:
I don't know honestly, the vue-multiselect class requires it. If I remove it then the whole thing crashes.

Reply:
Seems `name` is sufficient for the tracking purpose and it's safe to remove the `code` property.